content/en/agent/guide/how-do-i-uninstall-the-agent.md

```diff
@@ -197,44 +197,60 @@ sudo rm /Library/LaunchDaemons/com.datadoghq.agent.plist
 ```
 
 > This method removes the Agent, as well as all Agent configuration files.
----
+{{% /tab %}}
+{{< /tabs >}}
 
 ## Windows
-**Agent v6 & v7**
+
+{{< tabs >}}
+{{% tab "Agent v6 & v7" %}}
 
 There are two different methods to uninstall the Agent on Windows. Both methods remove the Agent, but do not remove the `C:\ProgramData\Datadog` configuration folder on the host.
 
-### 1. Remove programs
-  Uninstall the Agent using Add/Remove Programs.
+### Add or remove programs
+
+1. Press **CTRL** + **Esc** or use the Windows key to run Windows Search.
+1. Search for `add` and click **Add or remove programs**.
+1. Search for `Datadog Agent` and click **Uninstall**.
 
-### 2. Powershell
-  Alternatively, use one of the following Powershell commands. 
-  > **Note**: Some Agent versions may cause a forced reboot. To prevent this, add a no restart parameter using `REBOOT=ReallySuppress`:
+### PowerShell
 
+**Note:** You might need to enable WinRM to use the commands below.
+
+Use one of the following PowerShell commands to uninstall the Agent without rebooting:
 ```powershell
-start-process msiexec -Wait -ArgumentList ('/log', 'C:\\uninst.log', '/q', '/x', (Get-WmiObject -Class Win32_Product -Filter "Name='Datadog Agent'" -ComputerName .).IdentifyingNumber, 'REBOOT=ReallySuppress')
+start-process msiexec -Wait -ArgumentList ('/log', 'C:\uninst.log', '/q', '/x', (Get-CimInstance -ClassName Win32_Product -Filter "Name='Datadog Agent'" -ComputerName .).IdentifyingNumber, 'REBOOT=ReallySuppress')
 ```
 
 Using `/norestart`:
 
 ```powershell
-start-process msiexec -Wait -ArgumentList ('/log', 'C:\\uninst.log', '/norestart', '/q', '/x', (Get-WmiObject -Class Win32_Product -Filter "Name='Datadog Agent'" -ComputerName .).IdentifyingNumber)
+start-process msiexec -Wait -ArgumentList ('/log', 'C:\uninst.log', '/norestart', '/q', '/x', (Get-CimInstance -ClassName Win32_Product -Filter "Name='Datadog Agent'" -ComputerName .).IdentifyingNumber)
 ```
----
-**Agent v5**
+
+{{% /tab %}}
+
+{{% tab "Agent v5" %}}
 
 There are two different methods to uninstall the Agent on Windows. Both methods remove the Agent, but do not remove the `C:\ProgramData\Datadog` configuration folder on the host.
 
 > **Note**: For Agent < v5.12.0, it's important to uninstall the Agent with the **original account** used to install the Agent, otherwise it may not be cleanly removed.
 
-### 1. Remove programs
-  Uninstall the Agent using Add/Remove Programs.
+### Add or remove programs
+
+1. Press **CTRL** + **Esc** or use the Windows key to run Windows Search.
+1. Search for `add` and click **Add or remove programs**.
+1. Search for `Datadog Agent` and click **Uninstall**.
+
+### PowerShell
 
-### 2. Powershell
-  Alternatively, use the Powershell command below.
+**Note:** You might need to enable WinRM to use the commands below.
```

Comment:
It sounds like this may have been necessary -- could you confirm that `might` is the right word here? Just trying to keep in mind the RFC:
https://www.rfc-editor.org/rfc/rfc2119

Reply:
Yeah, good catch. I was hedging here cos I needed it when I ran the command 😅. I'll remove it.